client/src/Components/Common/Layout/Center.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
```

Comment:
This is a useful component to have. Are there other parts of the website we should wrap into this, looking at th new wireframes?

Reply:
yeah there are some (forget password, report, privacy & terms, review, profile, reply)
I think it would be better to work on them on a seperated issue, what do you think?

Reply:
Yeah I agree @RamyAlshurafa . If you can create a separate issue with your suggested pages to use it for (just give it a backlog tag) that would be great!